src/highdicom/sr/templates.py

```diff
@@ -1254,7 +1257,7 @@ def __init__(
             Description of the session
         measurements: Sequence[highdicom.sr.Measurement], optional
             Numeric measurements
-        qualitative_evaluations: Sequence[highdicom.sr.CodeContentItem], optional
+        qualitative_evaluations: Sequence[Union[highdicom.sr.CodeContentItem, highdicom.sr.QualitativeEvaluation]], optional
             Coded name-value pairs that describe measurements in qualitative
             terms
```

Comment:
A measurement (discrete or continuous numeric feature) and a qualitative evaluation (categorical feature) are conceptually two different things

```suggestion
            Coded name-value pairs that describe qualitative evaluations
```